site/app/entities/UserEntity.php

```diff
@@ -135,6 +135,7 @@ public function getDisplayInfo(): array {
         $out = [];
         $out["given_name"] = $this->user_preferred_givenname ?? $this->user_givenname;
         $out["family_name"] = $this->user_preferred_familyname ?? $this->user_familyname;
+        $out["full_name"] = $out["given_name"] . ' ' . $out["family_name"];
```

Comment:
I would just recreate `getDisplayFullName()` from User.php and put it in this class.